crates/amaru-ledger/src/summary/stake_distribution.rs

```diff
@@ -228,20 +231,34 @@ impl StakeDistribution {
                     return match pools.get_mut(&pool_id) {
                         None => false,
                         Some(pool) => {
-                            // NOTE: Governance deposits do not count towards the pools' stake.
-                            // They are only counted as part of the voting power.
+                            // NOTE: Governance deposits do not count towards the pools' active
+                            // stake used in rewards calculation. They are only counted as part of
+                            // the voting power & for consensus leader election.
                             active_stake += &account.lovelace;
-                            pool.stake += &account.lovelace;
+                            pool.rewards_stake += &account.lovelace;
+
+                            // NOTE: An astute reader might wonder why there's no total
+                            // 'pools_consensus_stake' which would correspond to the sum of all
+                            // pool's consensus stake. *Why* is hard to answer here, but *how* we
```

Comment:
```suggestion
                            // pools' consensus stake. *Why* is hard to answer here, but *how* we
```

Reply:
This comment feels bit coming out of the blue as there's no indication of any `total` in sight, perhaps it requires more context?